src/components/Toggle/Toggle.js

```diff
@@ -56,9 +56,13 @@ const Toggle = ({
       />
 
       <label className={`${prefix}--toggle__label`} htmlFor={id}>
-        <span className={`${prefix}--toggle__text--left`}>{labelA}</span>
+        <span className={`${prefix}--toggle__text--left`} aria-hidden="true">
```

Comment:
Based on the linked comment, do we need to be toggling which node has `aria-hidden="true"` depending on state?

Reply:
@joshblack Could you elaborate what you meant here...? To better explain what this PR is all about, @carmacleod wanted to avoid our on/off labels spoken by screen reader.

Reply:
I thought what she was referring to was that both pieces of text were being read, but it should only be the opposite of the currently selected value that is read. Like when toggle is off, it reads on, when the toggle is on, it reads off.

If I understand this diff correctly, it seems like this removes both on and off labels such that they can never be read by a screen reader.

Reply:
Gotcha, I think I get it now 😄 

The label is wrapped in the `<fieldset>` which communicates the label for the control. And then, since it's a checkbox, we don't need on/off because it's communicated by the checked state for the checkbox?

Reply:
> The label is wrapped in the `<fieldset>` which communicates the label for the control.

Ah ok, I see the "Toggle w/ label" example now.  :)

It's a bit odd to use fieldset, because folks would expect a group of controls, not just a label and one control. The screen reader reads as "Toggle w/ label **grouping**, checkbox [not] checked". 
So depending on the exact text in the legend, it could be confusing, i.e. probably "Expand tabs" might be ok (people might think that was a single toggle), but "Tab expansion" would not (because it might imply multiple controls).

Consider instead having a span (or div) for the label and use `aria-labelledby` to point to the span's id.

Of course, there are other ways to create a toggle or switch control without co-opting the control's label, and that way the label can be used for its proper purpose. :grinning:
i.e. See [Example 3](https://scottaohara.github.io/a11y_styled_form_controls/src/checkbox--switch/)

Reply:
Thanks @joshblack for clarifying and @carmacleod for the insights.

> Consider instead having a span (or div) for the label and use `aria-labelledby` to point to the span's id.

Double-checking - @carmacleod Do you mean ```<legend className={`${prefix}--label`}>{labelText}</legend>``` by "label" and ```<span className={`${prefix}--toggle__text--left`}>Off</span>```/```<span className={`${prefix}--toggle__text--right`}>On</span>``` by "span"?

Reply:
Sorry, I mean completely get rid of the fieldset/legend, and instead have:
```
  <div-or-span class="bx--label" id="toggle2label">Toggle w/ Label</div-or-span>
  <div class="bx--form-item">
      <input class="bx--toggle" id="toggle2" type="checkbox" aria-labelledby="toggle2label">
      <label class="bx--toggle__label" for="toggle2">
         <span class="bx--toggle__text--left" aria-hidden="true">Off</span>
         <span class="bx--toggle__appearance"></span>
         <span class="bx--toggle__text--right" aria-hidden="true">On</span>
      </label>
  </div>
```

where `div-or-span` is a div or a span (your choice) that has an id,
and the checkbox input uses `aria-labelledby` to point to that id.

Reply:
Got it - Thanks @carmacleod for clarifying! Replaced `<fieldset>`/`<legend>` with `<div>` with `aria-labelledby` reference from `<input>`.